lib/ext/heroku/command/addons.rb

```diff
@@ -251,7 +256,7 @@ def attach
       case response.status
       when 201
         display("done")
-        action("Setting #{response.body["name"]} vars and restarting #{app}") do
+        action("Setting #{response.body["name"]} and restarting #{app}") do
```

Comment:
@bjeanes +1 I meant to ask whether config vars not being present in that serialization was intentional or not, but it would be better to have them and use them in this output. I'll write up a card in Target Process. :boom: